### PR TITLE
ovs-performance: allow setting traffic macs

### PR DESCRIPTION
**Pull request by apconole**
Some hardware cannot operate in a promiscuous mode.  For this
hardware, we allow setting the dest mac address directly.

Signed-off-by: Aaron Conole <aconole@redhat.com>

### CODE REVIEW

File: ovs_performance.py [hunk 1]
```diff
@@ -123,6 +123,9 @@
 DEFAULT_STREAM_LIST               = '10, 1000, 10000, 100000, 1000000'
 DEFAULT_BRIDGE_NAME               = 'ovs_pvp_br0'
 DEFAULT_WARM_UP_TIMEOUT           = 360
+DEFAULT_DEST_MAC_ADDRESS          = '00:01:02:03:04:05'
```
Comment (chaudron) [inline ovs_performance.py]: Can you keep the current default mac addresses, which is "00:00:02:00:00:00"

File: ovs_performance.py [hunk 2]
```diff
@@ -982,37 +988,40 @@ def start_traffic_tx_on_vm(vm, nr_of_flows, packet_size):
               r"'rm -f ~/results.txt; " \
               r" nohup /bin/trafgen -c 3 -n 4 -- -p 1 --benchmark " \
               r"--flows-per-stream 1 --bursts-per-stream 1 --streams {3} " \
-              r"--src-mac 00:00:01:00:00:00 --dst-mac 00:00:02:00:00:00 " \
+              r"--src-mac 00:00:01:00:00:00 --dst-mac {5} " \
```
Comment (chaudron) [inline ovs_performance.py]: Should you not do SRC mac here also?

File: ovs_performance.py [hunk 3]
```diff
@@ -982,37 +988,40 @@ def start_traffic_tx_on_vm(vm, nr_of_flows, packet_size):
...
+                     config.dut_vm_password, nr_of_flows, packet_size,
+                     config.dest_mac_address)
     elif config.flow_type == 'L3':
         cmd = r"sshpass -p {2} ssh -o UserKnownHostsFile=/dev/null " \
               r"-o StrictHostKeyChecking=no -n {1}@{0} " \
               r"'rm -f ~/results.txt; " \
               r" nohup /bin/trafgen -c 3 -n 4 -- -p 1 --benchmark " \
               r"--flows-per-stream 1 --bursts-per-stream 1 --streams {3} " \
-              r"--src-mac 00:00:01:00:00:00 --dst-mac 00:00:02:00:00:00 " \
+              r"--src-mac 00:00:01:00:00:00 --dst-mac {5} " \
```
Comment (chaudron) [inline ovs_performance.py]: What about source mac?

File: ovs_performance.py (hunk 1 above)
Comment (apconole) [reply ovs_performance.py]: sure can - that was a mistake.

File: ovs_performance.py (hunk 3 above)
Comment (apconole) [reply ovs_performance.py]: will do.

File: ovs_performance.py (hunk 2 above)
Comment (apconole) [reply ovs_performance.py]: yes, will do.

**Comment by apconole** [comment]
I've pushed the requested changes.  Please review. :)

**Pull request closed without merging**